src/machines/commerce/__tests__/utils.ts

```diff
@@ -0,0 +1,269 @@
+import {rest} from 'msw'
+import {setupServer} from 'msw/node'
+import {
+  getPriceParams,
+  getStripeCheckoutParams,
+  priceFetcher,
+  eggheadPriceCheckUrl,
+  stripePriceCheckUrl,
+  stripeCheckoutSessionUrl,
+  checkoutSessionFetcher,
```

Comment:
`priceFetcher` and `checkoutSessionFetcher` are both services the commerce machine uses and they have logic related to fetching a stripe price vs a egghead sellable price.